nntrainer/tensor/bcq_tensor.cpp

```diff
@@ -292,6 +292,22 @@ std::vector<unsigned int> BCQTensor::argmax() const {
   return result;
 }
 
+std::vector<unsigned int> BCQTensor::argmin() const {
+  std::vector<unsigned int> result;
+  const uint32_t *data = (uint32_t *)getData();
+  size_t batch_size = batch();
+  size_t feature_len = dim.getFeatureLen();
+
+  result.resize(batch_size);
+
+  for (unsigned int b = 0; b < batch_size; b++) {
+    auto min_iter =
+      std::min_element(data + b * feature_len, data + (b + 1) * feature_len);
+    result[b] = std::distance(data, min_iter) - (b * feature_len);
+  }
```

Comment:
There are multiple identical multiplication operations in the code. 
Generally, addition takes less time than multiplication; 
would it be better to modify the code like the example?

```suggestion
  for (unsigned int b = 0, auto start_position = 0; b < batch_size; b++, start_position += feature_len) {
    auto min_iter =
      std::min_element(data + start_position, data + start_position + feature_len);
    result[b] = std::distance(data, min_iter) - start_position;
  }
```

Reply:
Thank you for the suggestion.
For the code, let's make the change in another PR; this `argmin()` op is defined in other tensor type as well. Thus, I've tried to match it with their implementation.
It would be better to update other tensors' argmin() implementation at the same time :)